packages/api-server/src/plugins/cache/cache-builder.ts

```diff
@@ -12,7 +14,8 @@ declare module 'fastify' {
 
 export default fp<FastifyOAuth2Options & FastifyJWTOptions>(async (fastify) => {
     const log = fastify.log as pino.Logger;
-    fastify.decorate('cacheBuilder', new CacheBuilder(log, fastify.config.ENABLE_CACHE));
+    const conn = await createConnection(getConnectionOptions());
+    fastify.decorate('cacheBuilder', new CacheBuilder(log, fastify.config.ENABLE_CACHE, conn));
```

Comment:
As I have mentioned above, we could just pass in an undefined `conn` indicating an disabled cache builder